README.md

```diff
@@ -72,6 +72,91 @@ would not be provable anyway. For tests that have an acceptable transaction `gas
 alter the latter to be `0xFFFFFFFF` (i.e. the maximum value fitting in a `u32`). If the runner manages to generate a valid witness / proof
 for this altered test, we log it as valid. If it fails, then we flag the test as ignored.
 
+## Coverage [zk_evm v0.4.0]
+
+The results below have been obtained against [zk_evm v0.4.0](https://github.com/0xPolygonZero/zk_evm/releases/tag/v0.4.0).
+
+All test vectors have been fetched from [ethereum/legacytests](https://github.com/ethereum/legacytests/tree/master) from
+commit [8077d24](https://github.com/ethereum/legacytests/commit/8077d241740de5a448e41156fd33740f562d3b56).
+
+The total number of tests below excludes:
+
+- tests with invalid block RLP encoding (from which all data is retrieved)
```

Comment:
Very nit: I personally didn't find `(from which all data is retrieved)` very clear (I might be tired, but I had to read it twice to understand). Maybe something like `(the encoding from which all the test data is retrieved)` would make it a bit clearer?

Reply:
Will update!